drgn_tools/task.py

```diff
@@ -161,3 +169,93 @@ def task_cpu(task: drgn.Object) -> int:
     if has_member(task, "cpu"):
         return task.cpu.value_()
     return task_thread_info(task).cpu.value_()
+
+
+def translate_nanoseconds(task: Object) -> str:
```

Comment:
This function is misnamed. It does not translate nanoseconds. It returns a string representation of the time since the task was last run. Could this be `task_lastrun2now_str()`?

Alternatively, you could instead have the function accept a number of nanoseconds, and convert it into a string representation of the time, and name it something like `format_nanosecond_duration()`